src/components/Person/Person.jsx

```diff
@@ -1 +1,34 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
+  const namePartner = person.sex === 'm' && person.isMarried
+    ? `${person.partnerName} is my wife`
+    : `${person.partnerName} is my husband`;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {' '}
+        {person.name}
+      </h2>
+
+      { person.age && (
+        <p className="Person__age">
+          I am
+          {' '}
+          {person.age}
+        </p>
+      )
+      }
+
+      { !person.isMarried ? (
+        <p className="Person__partner">I am not married</p>
+      )
+        : (
+          <p className="Person__partner">{namePartner}</p>
+        )
+  }
```

Comment:
as a suggestion, you can add check inside `p`
```suggestion
      <p className="Person__partner">
        {!person.isMarried ? 'I am not married' : namePartner}
      </p>
```